.github/workflows/deploy-cross-space-dapp.yaml

```diff
@@ -0,0 +1,45 @@
+name: Deploy Cross Space Dapp
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - dev
+
+jobs:
+  doploy-dapp-demo:
+    if: ${{ github.event.pull_request.merged == true && github.head_ref == '__release-pkg' }}
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [16.5]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v2
+        id: yarn-and-maven-cache # use this to check for `cache-hit` (`steps.yarn-and-maven-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: |
+            ~/.m2/repository
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            ./.shadow-cljs
+          key: ${{ runner.os }}-yarn-and-maven-${{ hashFiles('**/yarn.lock') }}-${{ hashFiles('./shadow-cljs.edn') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-and-maven-
+      - run: yarn install --immutable
+      - run: yarn workspace dapp-cross-space build
+      - run: cp packages/dapp-cross-space/CNAME packages/dapp-cross-space/dist/CNAME
+      - name: Deploy to cross-space.fluentwallet.dev
+        uses: dswistowski/surge-sh-action@v1
+        with:
+          domain: "cross-space.fluentwallet.dev"
```

Comment:
https://github.com/Conflux-Chain/helios/blob/d8c240d3d9643711e93412e1c9d5faaf1dbb7f6a/packages/rpcs/wallet_registerSiteMetadata/index.js#L49
maybe we should change the website URL when auto authed? or depend on the `ENV`

Reply:
dapp url: https://evm.fluentwallet.com